lib/controllers/todos.js

```diff
@@ -0,0 +1,35 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+const Todo = require('../models/Todo');
+const User = require('../models/User');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const data = await User.getById(req.user.id);
```

Comment:
this is totally fine but it does mean you're doing 2 requests when you could be doing one. If instead you define `Task.getUserTasks(req.user.id)` or something like that you can just get the tasks directly using a single query.